deployments/tools/export/lakefs_export.py

```diff
@@ -0,0 +1,117 @@
+import subprocess
+import argparse
+import os
+import shlex
+import time
+from datetime import datetime
+from string import Template
+
+LAKEFS_ACCESS_KEY = os.getenv('LAKEFS_ACCESS_KEY')
+LAKEFS_SECRET_KEY = os.getenv('LAKEFS_SECRET_KEY')
+LAKEFS_ENDPOINT = os.getenv('LAKEFS_ENDPOINT')
+S3_ACCESS_KEY = os.getenv('S3_ACCESS_KEY')
+S3_SECRET_KEY = os.getenv('S3_SECRET_KEY')
+
+SUCCESS_MSG = "Export completed successfully!"
+
+
+def create_rclone_conf_file():
+    with open('rclone.conf.template') as f:
+        src = Template(f.read())
+        vars = {'lakefs_access_key': LAKEFS_ACCESS_KEY, 'lakefs_secret_key': LAKEFS_SECRET_KEY, 'lakefs_endpoint': LAKEFS_ENDPOINT, 's3_access_key': S3_ACCESS_KEY, 's3_secret_key': S3_SECRET_KEY}
+        res = src.substitute(vars)
+    with open('rclone.conf', 'w+') as f:
+        f.write(res)
+    return
+
+
+def set_args():
+    parser = argparse.ArgumentParser(description='Process lakeFS export.')
+
+    parser.add_argument('Repo', metavar='repo', type=str, help='the name of lakeFS repository')
+    parser.add_argument('Dest', metavar='dest', type=str, help='path of destination')
+    parser.add_argument('--branch', metavar='branch', type=str, action='store', help='the relevant branch in the repository to export from')
+    parser.add_argument('--commit_id', metavar='commit', type=str, action='store', help='the relevant commit on the repository to export from')
+    parser.add_argument('--prev_commit_id', metavar='previous-commit', type=str, action='store', help='a previous commit reference to export only the diff between it and the head of the branch')
```

Comment:
Arguments that change behaviour of the command are often easier to understand if phrased imperatively.  Perhaps more along these lines:
```suggestion
    parser.add_argument('--prev_commit_id', metavar='previous-commit', type=str, action='store', help='if specified, export only the difference between this commit ID and the head of the branch')
```